tests/test_lookup_tools.py

```diff
@@ -36,30 +40,38 @@ def test_extractor_exceptions():
     except Exception as e:
         assert(e.args[0] == 'Cannot make an evaluator from unfinalized extractor!')
 
-def test_evaluator_exceptions():
+@pytest.mark.parametrize("awkwardlib", awkwardlibs)
+def test_evaluator_exceptions(awkwardlib):
     extractor = lookup_tools.extractor()
     extractor.add_weight_sets(["testSF2d scalefactors_Tight_Electron tests/samples/testSF2d.histo.root"])
 
     counts, test_eta, test_pt = dummy_jagged_eta_pt()
     test_eta_jagged = awkward.JaggedArray.fromcounts(counts, test_eta)
     test_pt_jagged = awkward.JaggedArray.fromcounts(counts, test_pt)
+    if awkwardlib == "ak1":
+        test_eta_jagged = awkward1.from_awkward0(test_eta_jagged)
+        test_pt_jagged = awkward1.from_awkward0(test_pt_jagged)
 
     extractor.finalize()
     evaluator = extractor.make_evaluator()
     
     try:
         test_out = evaluator["testSF2d"](test_pt_jagged, test_eta)
     except Exception as e:
-        assert(e.args[0] == 'do not mix JaggedArrays and numpy arrays when calling a derived class of lookup_base')
+        if awkwardlib == "ak0":
+            assert(e.args[0] == 'Do not mix JaggedArrays and other arrays when calling derived class of lookup_base')
+        if awkwardlib == "ak1":
+            assert(isinstance(e, ValueError))
 
-def test_evaluator_exceptions():
+def test_evaluate_noimpl():
     from coffea.lookup_tools.lookup_base import lookup_base
     try:
```

Comment:
`with pytest.raises(NotImplementedError)` might work too. Not sure what it tests though?